### ISSUE TITLE
Feature request - Control disabled devices

### ISSUE THREAD

**Issue by cappaji** [frgnca/AudioDeviceCmdlets#21]
Is it possible to add the cmdlet to return the device name of the disabled audio device ?

Currently if we execute  'Get-AudioDevice -Playback' after disabling the audio device we get below exception.
Get-AudioDevice : Element not found. (Exception from HRESULT: 0x80070490)
At line:1 char:1
+ Get-AudioDevice -Playback
+ ~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Get-AudioDevice], COMException
    + FullyQualifiedErrorId : System.Runtime.InteropServices.COMException,AudioDeviceCmdlets.GetAudioDevice

**Comment by frgnca** [comment]
Yes, currently AudioDeviceCmdlets only support enabled devices, but I think an optional -ShowDisabled parameter could be added.

**Comment by cappaji** [comment]
Thanks. It would be really great and helpful if the option -ShowDisabled is added.

**Comment by frgnca** [comment]
I opened the pull request #25 to try and add this feature in the next version.

**Comment by mrtbrt17** [comment]
Hello, Was this left abandoned?

**Comment by eabase** [comment]
Would definitely be useful to list disabled devices. Especially with all the on/off with using BT devices.
